### ISSUE TITLE
A11Y_Visual Studio Code Services_Docs User Guide_IntelliSense_ScreenReader:On navigating using letter T to the table it is read as table with 22 rows. 

### ISSUE THREAD

**Issue by v-jagansai** [microsoft/vscode-docs#5548]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#BM_Visual Studio Code Services_Aug2022;#Visual Studio Code Services;#Win11;#FTP;#DesktopWeb;#A11ySev2;#WCAG1.3.1;#NVDA;#JAWS;#STP;#Chrome;

## Environment Details:
Application Name: Visual Studio Code Services
URL: https://code.visualstudio.com/
Windows Version: Win11
Chrome Version: Version 103.0.5060.134 (Official Build) (64-bit)
NVDA Version:2021.3.5,JAWS

## Repro Steps:
1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix
2. Launch the application Url : https://code.visualstudio.com/
3. Navigate to "Docs" link.
4. From left navigation, select "User Guide".
5. Navigate to "IntelliSense" link. 
6. Start Screen Reader .
7. Tab to table presented in the screen .
8. Observe that the Screen reader is announcing the correct rows and tables.
## Actual:
On navigating using letter T to the table it is read as table with 22 rows, On navigating row three using CTRL + ALT+ Arrow keys it is read as Row four.
Same issue is seen with Chrome+JAWS.

## Expected:
On navigating using letter T to the table it should be read as table with 21 rows as visually displayed, Row three should be read as three instead of four.

## User Impact:
Screen reader will not get correct count of the rows.

## Attachment
![Presentation4 gif latest](https://user-images.githubusercontent.com/109065315/184306363-f0aec639-b90b-44b2-a030-e0b4dc15ca45.gif)
![22 rows](https://user-images.githubusercontent.com/109065315/184351325-c159e4df-dce5-4eef-a60f-c5f047024bca.gif)

**Comment by gregvanl** [comment]
There is an empty first row in the table. Can likely be fixed by providing column heading text

**Comment by prabhudevu** [comment]
![5548_Issue fixed](https://user-images.githubusercontent.com/98821792/216557573-d5ef156d-32e5-4550-9019-05a191cb1130.png)

I have verified the issue in the Env: https://code.visualstudio.com/docs/editor/intellisense Issue is fixed. closing the bug

GithubTags: #Closed;
